### PR TITLE
useSnackbar hook

### PR DESCRIPTION
**Pull request by leovalette**
The aim of this hook is to handle concurrency display of snackbars and to make the Snackbars easier to use. Here you just need to call the hook which returns 2 things :

- openSnackbar : a method to open a snackbar, call it with the message you want to display and its severity
- snackbarParams : the paramaters you'll need to give to the Snackbar component

### CODE REVIEW

File: packages/yourloops/pages/profile/profile.tsx [hunk 1]
```diff
@@ -281,14 +277,25 @@ export const ProfilePage: FunctionComponent = () => {
...
+    handleUserUpdate,
+  ]);
 
   const onCancel = (): void => history.goBack();
-  const onCloseAlert = (): void => setApiReturnAlert(null);
 
   return (
     <Fragment>
       <ProfileHeader />
+      <Snackbar snackbarParams={snackbarParams} />
```
Comment (Nas84) [inline packages/yourloops/pages/profile/profile.tsx:298]: Could you rename the first snackbarParams to params ?

File: packages/yourloops/components/utils/snackbar.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  snackbarParams: {
+    apiAlert: ApiAlert;
+    removeAlert: (apiAlertId: ApiAlert["id"]) => void;
+  };
+}
+
+export const Snackbar = ({ snackbarParams: { apiAlert, removeAlert } }: SnackbarsProps): JSX.Element | null => {
+  const onCloseAlert = useCallback(
+    (id: ApiAlert["id"]) => (_: React.SyntheticEvent | MouseEvent, reason?: string) => {
+      if (reason === "clickaway") {
```
Comment (Nas84) [inline packages/yourloops/components/utils/snackbar.tsx:46]: a comment here to explain why we need test the reason can be usefull

**Review by Nas84 — CHANGES_REQUESTED** [review]

**Review by Nas84 — APPROVED** [review]
LGTM

**Review by yvon-dblg — COMMENTED** [review]
Not sure: But maybe for the already implemented snackbar (I'm thinking of the one in pages/hcp), you could use your hook ?

**Pull request merged**